### ISSUE TITLE
Version 0.5 Updates

### ISSUE THREAD

**Issue by israel-dryer** [israel-dryer/ttkbootstrap#82]
I just pushed [quite a few changes](https://github.com/israel-dryer/ttkbootstrap/commits/version-0.5) to GitHub for Version 0.5. I plan on pushing these changes to PyPI tomorrow afternoon unless there are any objections.

## ttkbootstrap
- changed theme file from json to py
- remove dependency on external fonts
- remove dependency on importlib_resources
- remove default font (relies on os defaults)
- updated all style colors
- added highlight effects on dark theme Entry type widgets (this is now possible because of the darker inputbg color)
- added 'morph' theme
- changed the drawing of the radio and checkbutton to the same as the version 1.0 (except for no scaling)

![ttkbootstrap](https://user-images.githubusercontent.com/42584742/144358970-5a7542c7-0a0e-40eb-8326-9bbe42edc8a0.gif)

## ttkcreator
- removed theme splash screen
- added base theme combobox to select starting colors (have one more tweak on this to make)
- added import an export buttons which import and export to the themes_custom.py file

<img width="378" alt="ttkcreator" src="https://user-images.githubusercontent.com/42584742/144358905-052e7994-654b-4f55-aa15-9c50cba70779.png">

**Comment by daniilS** [comment]
Apart from the three issues I submitted PRs for, it looks ready to be pushed to PyPI!

**Comment by israel-dryer** [comment]
@daniilS

Ok. Had to make a few more tweaks, lol. I'm regretting going down this path now... but too late. Oh well.

I changed the ttk creator to the version 1.0 style... and this gave me a much better look at the styles, which needed some more tweaks. I updated the selectbg color on the dark themes.. which makes a big difference. I also fixed the horizontal scrollbar arrows, which were wonky. And then a few other small things.

Here are the new theme colors... slightly modified.

![ttkbootstrap](https://user-images.githubusercontent.com/42584742/144551612-bc17e2f0-9c54-4215-8025-909e7858adc5.gif)

And, here is the new ttk creator app... I actually used it to figure out the correct selectbg colors for the dark themes. very useful.

<img  alt="ttkcreator" src="https://user-images.githubusercontent.com/42584742/144551703-4de795e1-bedd-4411-8ce8-3cb6f94b1009.png">
